### PR TITLE
Card Scheduling PS-596 (minDate/maxDate bug fix)

### PR DESCRIPTION
**Pull request by shyankashani**

### CODE REVIEW

File: addon/mixins/pikaday.js [hunk 1]
```diff
@@ -128,7 +129,7 @@ export default Ember.Mixin.create({
 
       // If the current date is greater than maxDate we set date to maxDate
       run.schedule('sync', () => {
-        if (value > maxDate) {
+        if (!allowBlank && value > maxDate) {
```
Comment (thomaswrenn) [inline addon/mixins/pikaday.js]: Can you foresee any use case where we might want to independently control min and max allowBlank-ness? I don't think so. So seems good to me.

**Review by thomaswrenn — COMMENTED** [review]
Nice!

Definitely should add and execute a manual test plan.

And should look at the way they test and see if it's easy to add tests for this or if they test that throughly.

Let me know when you add the manual test plan and if it doesn't make sense to add automated tests.

File: addon/mixins/pikaday.js [hunk 2]
```diff
@@ -111,15 +112,15 @@ export default Ember.Mixin.create({
 
       // If the current date is lower than minDate we set date to minDate
       run.schedule('sync', () => {
-        if (value < minDate) {
+        if (!allowBlank && value < minDate) {
```
Comment (shyankashani) [inline addon/mixins/pikaday.js]: Update comment

File: addon/mixins/pikaday.js [hunk 3]
```diff
@@ -102,7 +103,7 @@ export default Ember.Mixin.create({
   },
 
   setMinDate: function() {
-    const { pikaday, minDate, value } = getProperties(this, [ 'pikaday', 'minDate', 'value' ]);
+    const { pikaday, minDate, value, allowBlank } = getProperties(this, [ 'pikaday', 'minDate', 'value', 'allowBlank' ]);
```
Comment (shyankashani) [inline addon/mixins/pikaday.js]: Remove the allowBlank property altogether. If the `value` is null, don't implement the defaulting to `minDate`/`maxDate`.

**Pull request closed without merging**